test/tests/mesh_modifiers/break_mesh_by_blocks/3D/tests

```diff
@@ -69,6 +69,19 @@
   issues = '#11775'
 [../]
 
+[./3DPolyCrystalBreakMesh_test]
+  ###  compare manual vs automatic mesh split
+  type = 'Exodiff'
+  input = 'TestBreakMesh_3D_polycrystal.i'
+  cli_args = '--mesh-only'
+  exodiff = 'TestBreakMesh_3D_polycrystal_in.e'
+  mesh_mode = REPLICATED
+  recover = false
+  requirement = "The MeshModifier should break the mesh into multiple unconnected grains and create an interface on all grain boundaries"
```

Comment:
This requirement is duplicated between two tests which is pretty bad if we are building a requirements document from these in an automated fashion. You created two tests so clearly you are satisfying 2 different requirements. This one is for 3D specifically and the other is for 2D. It's worth mentioning that in your requirements sentence. Otherwise this is great! Thanks!